test/extended/testdata/build-extended/bc-scripts-by-url.json

```diff
@@ -0,0 +1,39 @@
+{
+  "kind": "BuildConfig",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "java-extended-build-from-url"
+  },
+  "spec": {
+    "source": {
+      "git": {
+        "uri": "https://github.com/php-coder/java-maven-hello-world.git"
```

Comment:
can you at least use this one which i think we already use elsewhere, so we can move off it across the board when we do?
https://github.com/bparees/javaee7-hol
(if we don't use that one elsewhere, see if you can find one we do)